src/chatkit.ts

```diff
@@ -394,6 +424,36 @@ export default class Chatkit {
     });
   }
 
+  private updatePermissionsForRole(
+    roleName: string,
+    scope: string,
+    permissionsToadd: Array<string>,
+    permissionsToRemove: Array<string>,
+  ): Promise<any> {
+    if (len(permissionsToadd) == 0 && len(permissionsToRemove) == 0) {
+      throw new Error(`Either permissionsToAdd or permissionsToRemove is required`)
+    }
+
+    var body: object = {}
```

Comment:
Don't type as `object`, type as `any` or create an interface like at the top of the file, e.g.:

```typescript
export interface GetRoomMessagesOptions {
  initial_id?: string;
  direction?: string;
  limit?: number;
}
```